lib/utils.js

```diff
@@ -120,18 +120,85 @@ const getNameString = (files, itemName, regex) => {
   return nameString;
 };
 
+/**
+ * Gets all the open pull requests
+ * @param {import('probot').Context} context
+ */
+const getAllOpenPullRequests = async (context) => {
```

Comment:
We should also have separate test cases for these new functions even if they are being covered by any of the modules.

Reply:
Done.